frontend/src/components/Footer/index.jsx

```diff
@@ -0,0 +1,19 @@
+import "./Footer.css";
+import Box from "@mui/material/Box";
+
+function Footer() {
+  return (
+    <div>
+      <Box
+        display="flex"
+        justifyContent="center"
+        alignItems="center"
+        className="container"
```

Comment:
I suggest to avoid such a generic class name to avoid side effects on other parts of the app (if another component is given the class name "container", it will also inherit the CSS specific to this footer class)